lib/connection.js

```diff
@@ -29,17 +29,19 @@ function Connection(options) {
   this.apiCallCount = 0;
 
   // Convenient methods, matches that of restler's
-  _.each(restlerMethodArgCount, function(count, method) {
+  _.forEach(restlerMethodArgCount, async (count, method) => {
```

Comment:
@hac-madkudu I have kept backwards compatibility to older versions of node in this library. If we introduce `async/await`, this will a breaking change for node < 8, therefore the version need to be a major version.

Reply:
Given that this is the only change, preference would be to stay away from async/await for now, until we decide we want to drop support for node 4.